beginner_source/quickstart/data_quickstart_tutorial.py

```diff
@@ -0,0 +1,186 @@
+"""
+Datasets & Dataloaders
+===================
+"""
+
+#################################################################
+# Getting Started With Data in PyTorch
+# -----------------
+#
+# Before we can even think about building a model with PyTorch, we need to first learn how to load and process data. Data can be sourced from local files, cloud datastores and database queries. It comes in all sorts of forms and formats from structured tables to image, audio, text, video files and more.
+#
+
+###############################################################
+# .. figure:: /_static/img/quickstart/typesdata.png
+#    :alt: typesdata
+#
+# Different data types require different python libraries to load and process such as `openCV <https://opencv.org/>`_ and `PIL <https://pillow.readthedocs.io/en/stable/reference/Image.html>`_ for images, `NLTK <https://www.nltk.org/>`_ and `spaCy <https://spacy.io/>`_ for text and `Librosa <https://librosa.org/doc/latest/index.html>`_ for audio.
+#
+# If not properly organized, code for processing data samples can quickly get messy and become hard to maintain. Since different model architectures can be applied to many data types, we ideally want our dataset code to be decoupled from our model training code. To this end, PyTorch provides a simple Datasets interface for linking managing collections of data.
+#
+# A whole set of example datasets such as Fashion MNIST that implement this interface are built into PyTorch extension libraries. These are useful for benchmarking and testing your models before training on your own custom datasets.
+#
+#  You can find some of them below.
+#
+#  - `Image Datasets <https://pytorch.org/docs/stable/torchvision/datasets.html>`_
+#  - `Text Datasets  <https://pytorch.org/text/datasets.html>`_
+#  - `Audio Datasets <https://pytorch.org/audio/datasets.html>`_
+#
+
+#################################################################
+# Iterating through a Dataset
+# -----------------
+#
+# Once we have a Dataset we can index it manually like a list `clothing[index]`.
```

Comment:
Did you intend for this section to come after you explain Datasets?

Reply:
Yes my intent is to first explain how the reader can train on their own data and then point them to the reference examples.